docs/configuration.md

```diff
@@ -67,6 +67,7 @@ Unless you need to set a non-default value, it is recommended to only populate o
 | `FEE_HISTORY_MAX_RESULTS`                   | "10"                     | The maximum number of results to returns as part of `eth_feeHistory`.                                                                                                                                                                                                                                  |
 | `ETH_FEE_HISTORY_FIXED`                     | "true"                   | Flag to set if eth_feeHistory should return a fixed fee for the set of results.                                                                                                                                                                                                                        |
 | `ETH_POPULATE_SYNTHETIC_CONTRACT_RESULTS`   | "true"                   | Flag to set if the relay should populate the contract results for synthetic contracts.                                                                                                                                                                                                                 |
+| `GAS_PRICE_PERCENTAGE_BUFFER`               | "0"                      | The additional buffer that adds a percentage on top of the calculated network gasPrice. This may be an option to fix INSUFFICIENT_TX_FEE error thrown due recently changes in exchange rates.                                                                                                          |
```

Comment:
I feel that the part "This may be an option to fix the INSUFFICIENT_TX_FEE error caused by recent changes in exchange rates." shouldn't be included because it doesn't seem relevant in a global context.

Reply:
I think it's a fair mention that informs why someone might use it.
Maybe it can be better worded as 
```suggestion
| `GAS_PRICE_PERCENTAGE_BUFFER`               | "0"                      | The additional buffer that adds a percentage on top of the calculated network gasPrice. This may be used by operators to reduce the chances of `INSUFFICIENT_TX_FEE` errors experienced by users caused by minor fluctuations in the exchange rate.                                                                                                          |
```

Reply:
Done.

Reply:
Ah with the new version, it sounds more confident!